libgit/repo.go

```diff
@@ -57,6 +57,19 @@ var CommitSentinelValue *object.Commit
 // test.
 var repoNameRE = regexp.MustCompile(`^([a-zA-Z0-9][a-zA-Z0-9_\.-]*)$`)
 
+// CommitsByRefName represents a map of reference names to a list of commits.
+type CommitsByRefName map[plumbing.ReferenceName][]*object.Commit
+
+// GitPushType is a libgit export of keybase1.GitPushType.
+type GitPushType keybase1.GitPushType
+
+const (
+	GitPushType_DEFAULT      GitPushType = GitPushType(keybase1.GitPushType_DEFAULT)
+	GitPushType_CREATEREPO   GitPushType = GitPushType(keybase1.GitPushType_CREATEREPO)
+	GitPushType_DELETEBRANCH GitPushType = GitPushType(keybase1.GitPushType_DELETEBRANCH)
```

Comment:
Are you intentionally not sending this one yet?  Need a TODO?

Reply:
It needs to be in the ref section, so I'm leaving it for now.